webui/src/layout/Sidebar/Sidebar.tsx

```diff
@@ -27,7 +30,7 @@ const Sidebar: React.FC<SidebarProps> = ({ isCollapsed, toggleSidebar }) => {
     },
     {
       path: "/Settings",
-      icon: <Cogwheel />,
+      icon: <CogWheelIcon />,
```

Comment:
Should these be using `ReactSVG` now for consistency? Isn't this just going to output the `src` path now?